media/img/home/voices/promos/webmaker/webmaker-teacher.svg

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" [
+	<!ENTITY ns_flows "http://ns.adobe.com/Flows/1.0/">
```

Comment:
Do we need the `DOCTYPE` etc. I reckon we can remove everything to just before the open `<svg....`